src/monitor/monitor.c

```diff
@@ -37,10 +37,12 @@
 #include <sys/prctl.h>
 
 #include "util/sss_ini.h"
+#include "util/sss_ptr_hash.h"
 #include "confdb/confdb.h"
 #include "confdb/confdb_setup.h"
 #include "db/sysdb.h"
 #include "sss_iface/sss_iface_async.h"
+#include "sbus/sbus_private.h"
```

Comment:
Hi, could you please add a function to sbus server that will lookup a connection for you instead of touching private parts?

Reply:
Nice you mentioned this because, while I was making these changes, I found a function already doing this. So I'm using it now.